.github/workflows/post_aura_gauge_votes.yaml

```diff
@@ -1,25 +1,46 @@
 name: Post vlAURA snapshot votes to voter multisig and send to the vote relayer
 
 on:
-  workflow_dispatch:
-    inputs:
-      week-string:
-        description: 'Week string that votes are are being posted. should be YYYY-W##'
-        required: true
+  push:
```

Comment:
It wouldn't hurt to make another action for workflow dispatch or have a way to trigger this otherwise if the push doesn't work/to debug.  Not critical tho, can happen later.